reports/test_filters.py

```diff
@@ -17,9 +17,11 @@ class ReportFilterMixin(object):
 
```

Comment:
I have no idea why this file isn't in the `tests` directory. I'll move it there.